packages/demos/src/history-service/index.test.ts

```diff
@@ -2,24 +2,64 @@
  * @jest-environment puppeteer
  */
 
+// tslint:disable:no-non-null-assertion
+
 import {Server} from 'http';
 import {AddressInfo} from 'net';
+import {ElementHandle} from 'puppeteer';
 import {parse} from 'url';
 import {startServer} from '../start-server';
 import webpackConfigs from './webpack-config';
 
 jest.setTimeout(60000);
 
-async function getInputValue(selector: string): Promise<string | undefined> {
-  const element = await page.$(selector);
+class HistoryConsumerUI {
+  public static readonly a = new HistoryConsumerUI('a');
+  public static readonly b = new HistoryConsumerUI('b');
+
+  private constructor(private readonly prefix: 'a' | 'b') {}
```

Comment:
Technically, based on your usage, this would be called a `suffix`. But we could also call it `specifier` for example.